packages/terra-site/src/site.scss

```diff
@@ -64,33 +64,19 @@
     background-color: #80c7ee;
     border-bottom: 3px solid #047cc0;
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-end;
     overflow: hidden;
     padding: 0.5rem 1.25rem;
     position: relative;
     width: 100%;
   }
 
-  .site-utility {
-    display: flex;
-    flex: 1 1 auto;
-    justify-content: flex-end;
-    position: relative;
-    width: 100%;
+  .site-header > div:first-child{
+    flex: 3 0 auto;
   }
 
-  .site-locale,
-  .site-theme {
-    align-self: center;
-    flex: 0 0 auto;
-    padding-right: 1.25rem;
-    position: relative;
-  }
-
-  .site-bidi {
-    align-self: center;
-    flex: 0 0 auto;
-    position: relative;
+  .site-header > div[class*="divider"][class*="face-up-item"]{
+    border-right: none;
```

Comment:
Why are we hiding this?

Reply:
We don't necessarily have not. Here is what it looks like without styling:

<img width="909" alt="screen shot 2017-10-09 at 2 16 17 pm" src="https://user-images.githubusercontent.com/14099737/31354569-77ffd8a4-acfc-11e7-9239-cb99f97cc2ce.png">

vs with:

<img width="926" alt="screen shot 2017-10-09 at 2 16 53 pm" src="https://user-images.githubusercontent.com/14099737/31354596-8a6623fe-acfc-11e7-91b0-779c00e30ae6.png">

Reply:
This has no affect on the rolled up divider

Reply:
It feels like we are breaking our own rules if we are adding custom CSS to change the behavior / styles of our own Component. 

If there is value in hiding the divider here and showing it in the menu it might be worth talking with UX.